src/features/tasks/tasksSlice.js

```diff
@@ -254,6 +266,78 @@ export const tasksSlice = createSlice({
       state.loadingStates.cameraSerialNumber.taskId = null;
       state.loadingStates.cameraSerialNumber.errors.splice(index, 1);
     },
+
+    // delete images
+
+    deleteImagesStart: (state) => {
+      let ls = state.loadingStates.deleteImages;
+      ls.taskId = null;
+      ls.isLoading = true;
+      ls.errors = null;
+    },
+
+    deleteImagesUpdate: (state, { payload }) => {
+      state.loadingStates.deleteImages.taskId = payload.taskId;
+    },
+
+    deleteImagesSuccess: (state) => {
+      let ls = state.loadingStates.deleteImages;
+      ls.taskId = null;
+      ls.isLoading = false;
+      ls.errors = null;
+    },
+
+    deleteImagesFailure: (state, { payload }) => {
+      let ls = state.loadingStates.deleteImages;
+      ls.isLoading = false;
+      ls.errors = [payload.task.output.error];
```

Comment:
Do you mind also wiring up these errors and the `deleteImagesByFilters` errors in [`ErrorToast.jsx`](https://github.com/tnc-ca-geo/animl-frontend/blob/main/src/components/ErrorToast.jsx)?

Reply:
Any suggestion on testing this?

Reply:
The way you described in the meeting is how I'd go about it (intentionally testing out throwing errors at various points in the animl-api stack and making sure they are displayed to the user).